test/utils/test_dropout.py

```diff
@@ -59,3 +64,33 @@ def test_dropout_edge():
     out = dropout_edge(edge_index, force_undirected=True)
     assert out[0].tolist() == [[0, 1, 1, 2], [1, 2, 0, 1]]
     assert out[1].tolist() == [0, 2, 0, 2]
+
+
+def test_dropout_path():
```

Comment:
Run this test only when `torch_cluster` is available.
`@withPackage('torch_cluster')`